.docker/build/build.sh

```diff
@@ -34,12 +34,16 @@ function vale_test() {
 }
 
 function hacl_test() {
+    make_target="ci"
+    if [[ $target == "hacl-ci-build" ]]; then
+        make_target="vale-fst all-unstaged test-c-staged test-c"
```

Comment:
We talked about a new make target. I'm fine with adding one, but @msprotz asked if we could do without. Which ever is less likely to introduce breakage in future for any build is preferable imho.

Reply:
I prefer a Makefile target mainly because it's easier to maintain when target definitions and names are in a single place. 
Also, the `hacl-star` Makefile uses variables that modify targets, including `FSTAR_DEPEND_FLAGS="--warn_error +285"` that isn't included here.